go/chat/s3/s3.go

```diff
@@ -133,7 +136,7 @@ var DefaultAttemptStrategy = AttemptStrategy{
 }
 
 // New creates a new S3.  Optional client argument allows for custom http.clients to be used.
-func New(signer Signer, region Region, client ...*http.Client) *S3 {
+func New(signer Signer, region Region, env *libkb.Env, client ...*http.Client) *S3 {
```

Comment:
Why not `libkb.GlobalContext` instead?

Reply:
BTW, make sure you check out this doc: https://github.com/keybase/client/blob/master/go/doc/context.md

Reply:
I agree I think `libkb.GlobalContext` would be better than `libkb.Env`.

Reply:
Changed to `libkb.GlobalContext`. I winded `Env` through there since it required slightly fewer changes in order to do, but :+1: if we're okay with slightly more changes to the s3 code.